## Q211: 

C:\Users\gabim\AppData\Local\Programs\Python\Python37-32\python.exe C:/Users/gabim/PycharmProjects/Lab2_CC/WordCountTensorFlow_1.py
[nltk_data] Downloading package punkt to
[nltk_data]     C:\Users\gabim\AppData\Roaming\nltk_data...
[nltk_data]   Package punkt is already up-to-date!
[('the', 1343), (',', 1251), ('.', 810), (')', 638), ('(', 637), ('of', 586), ('to', 491), ('a', 470), (':', 454), ('in', 417)]
3383

Process finished with exit code 0
